rustler_mix/lib/rustler/compiler/config.ex

```diff
@@ -2,31 +2,20 @@ defmodule Rustler.Compiler.Config do
   @moduledoc false
 
   @type rust_version :: :stable | :beta | :nightly | binary()
-  @type cargo :: :system | {:rustup, rust_version()} | {:bin, Path.t()}
   @type crate :: binary()
   @type default_features :: boolean()
   @type env :: [{binary(), binary()}]
   @type features :: [binary()]
   @type mode :: :debug | :release
   @type load_data :: term()
-  @type path :: Path.t()
 
-  defstruct cargo: :system,
-            crate: nil,
-            default_features: true,
-            env: [],
-            external_resources: [],
-            features: [],
+  defstruct crate: nil,
             lib: true,
             load_data: 0,
-            load_from: nil,
+            load_path: nil,
```

Comment:
The intention for this was to allow pre-compiled artifacts to be loaded from your own `priv` dir.

Reply:
I know, maybe we should discuss this in a bit more detail.